include/onnxruntime/core/graph/graph.h

```diff
@@ -1016,6 +1022,9 @@ class Graph {
   /** Initialize function body for the given node */
   void InitFunctionBodyForNode(Node& node);
 
+  /** Gets Model local functions from the root/parent graph.*/
+  std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*>& GetModelLocalFunctions();
+
```

Comment:
is the caller permitted to change the map? if not it should return const std::unordered_map<...>